source/declarators.tex

```diff
@@ -2110,7 +2110,7 @@
     attribute-specifier-seq\opt decl-specifier-seq\opt declarator virt-specifier-seq\opt{} \terminal{ = default ;}
 \end{ncbnf}
 
-is called an \grammarterm{explicitly-defaulted} definition.
+is called an \term{explicitly-defaulted} definition.
 A function that is explicitly defaulted shall
```

Comment:
There's already a `\indextext{definition!function!explicitly-defaulted}` ~7 lines above, which is why I went with `\term`.